app/src/test/java/com/example/kevinschweitzer/myapplication/CalculatorModelTest.java

```diff
@@ -0,0 +1,88 @@
+package com.example.kevinschweitzer.myapplication;
+
+import com.example.kevinschweitzer.myapplication.mvp.model.CalculatorModel;
+import com.example.kevinschweitzer.myapplication.util.operators.DivisionOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.MinusOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.MultiplicationOperator;
+import com.example.kevinschweitzer.myapplication.util.operators.PlusOperator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+@RunWith(RobolectricTestRunner.class)
+public class CalculatorModelTest {
+
```

Comment:
You should to create another method with the annotation @before and create a unique instante of  CalculatorModel. And use this instance in all test